hummingbot/client/settings.py

```diff
@@ -108,9 +108,9 @@ def get_connector_spec(connector_name: str, chain: str, network: str) -> Optiona
 
     @staticmethod
     def get_connector_spec_from_market_name(market_name: str) -> Optional[Dict[str, str]]:
-        vals = market_name.split("_")
-        if len(vals) >= 3:
-            return GatewayConnectionSetting.get_connector_spec(vals[0], vals[1], "_".join(vals[2:]))
+        vals = market_name.rsplit("_", 2)
```

Comment:
```suggestion
        vals = market_name.split("_", 2)
```
I believe this is what you intend.

Reply:
No, I am using `rsplit`

Reply:
`rsplit` would cause issues.
i.e. rpslipt on `ethereum_mainnet_connector_name` will give `[ethereum_mainnet, connector, name]` whereas we need `[ethereum, main net, connector_name]`

Reply:
If you study existing code, you'll see it's equivalent to `split("_", 2)`

Reply:
Yes I would agree with you if the `market_name` is provided as such `chain_network_connector_name`. However, we are currently `market_name` is return in the following format `connector_name_chain_network`. i.e. `mad_meerkat_cronos_mainnet`. As such using `rsplit` would return `['mad_meerkat', 'cronos', 'mainnet']`

Reply:
`connector_name_chain_network` is right. Sorry for the confusion there.
I didn't give a proper example there.
Arbitrum is sent as `arbitrum_one` from the gateway. So the example I intended to give was `uniswap_ethereum_arbitrum_one`.
Now that mad_meerkat is also sent with a '_', I guess there's a problem that has to be addressed because we can't go on to keep splitting like that.
I suggest we discuss during or after standup. LMK what you think.